website/views.py

```diff
@@ -105,7 +106,11 @@ def post(self, request):
                 user.first_name = first_name
                 user.last_name = last_name
                 if len(request.FILES) != 0:
-                    profile.photo = photo
+                    if (photo.size <= settings.MAX_FILE_ULOAD):
+                        profile.photo = photo
+                    else:
+                        messages.warning(request, ('File size is too big.'))
```

Comment:
First, is it possible to avoid code duplication and reuse the validator? Second, if not then the error message could be better (use the one from the validator).